commands/remove_test.go

```diff
@@ -22,10 +24,60 @@ func Test_remove(t *testing.T) {
 
 	resetForTest()
 
+	stdOut := test.CaptureStdout(func() {
+		faasCmd.SetArgs([]string{
+			"remove",
+			"--gateway=" + s.URL,
+			"test-function",
+		})
+		faasCmd.Execute()
+	})
+
+	expectedStdOut := "Deleting: test-function."
+	if found, err := regexp.MatchString(`(?m:`+expectedStdOut+`)`, stdOut); err != nil || !found {
+		t.Fatalf("Output is not as expected:\nGot: %s\nExpected: %s", stdOut, expectedStdOut)
+	}
+}
+
+func Test_remove_stackYAML(t *testing.T) {
+	s := test.MockHttpServer(t, []test.Request{
+		{
+			Method:             http.MethodDelete,
+			Uri:                "/system/functions",
+			ResponseStatusCode: http.StatusOK,
+		},
+	})
+	defer s.Close()
+
+	resetForTest()
+	funcName := "stack"
+	yamlFile = "stack.yml"
+
+	// Cleanup the created directory
+	defer func() {
+		os.RemoveAll(funcName)
+		os.Remove(yamlFile)
+	}()
+
 	faasCmd.SetArgs([]string{
-		"remove",
+		"new",
 		"--gateway=" + s.URL,
-		"test-function",
+		"--lang=" + "Dockerfile",
+		"stack",
 	})
 	faasCmd.Execute()
+
+	stdOut := test.CaptureStdout(func() {
+		faasCmd.SetArgs([]string{
+			"remove",
+			"--gateway=" + s.URL,
+			"test-function",
+		})
+		faasCmd.Execute()
+	})
+
+	expectedStdOut := "Deleting: test-function."
```

Comment:
why do we need a variable `expectedStdOut`?

Reply:
It helps for printing out the error message.